Procfile.dev

```diff
@@ -1,2 +1,2 @@
 web: bin/rails server -p 3000
-css: bin/rails tailwindcss:watch
+css: yarn build:css --watch
```

Comment:
changing to allow more customisability (see script that build:css runs in package.json)